modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view_table.jsp

```diff
@@ -0,0 +1,239 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/admin/init.jsp" %>
+
+<%
+long kbFolderClassNameId = PortalUtil.getClassNameId(KBFolderConstants.getClassName());
+
+long parentResourceClassNameId = ParamUtil.getLong(request, "parentResourceClassNameId", kbFolderClassNameId);
+
+boolean kbFolderView = parentResourceClassNameId == kbFolderClassNameId;
+
+KBArticleURLHelper kbArticleURLHelper = new KBArticleURLHelper(renderRequest, renderResponse);
+
+KBAdminManagementToolbarDisplayContext kbAdminManagementToolbarDisplayContext = new KBAdminManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, renderRequest, renderResponse, portletConfig);
+
+SearchContainer<Object> kbAdminManagementToolbarDisplayContextSearchContainer = kbAdminManagementToolbarDisplayContext.getSearchContainer();
+
+KBArticleViewDisplayContext kbArticleViewDisplayContext = new KBArticleViewDisplayContext(request, liferayPortletRequest, liferayPortletResponse, renderResponse);
+%>
+
+<liferay-ui:search-container
+	id="kbObjects"
+	searchContainer="<%= kbAdminManagementToolbarDisplayContextSearchContainer %>"
+>
+	<liferay-ui:search-container-row
+		className="Object"
+		modelVar="object"
+	>
+		<c:choose>
+			<c:when test="<%= object instanceof KBFolder %>">
+
+				<%
+				KBFolder kbFolder = (KBFolder)object;
+
+				row.setData(
+					HashMapBuilder.<String, Object>put(
+						"actions", StringUtil.merge(kbAdminManagementToolbarDisplayContext.getAvailableActions(kbFolder))
+					).build());
+
+				row.setPrimaryKey(String.valueOf(kbFolder.getKbFolderId()));
+				%>
+
+				<liferay-ui:search-container-column-text
+					name="title"
+				>
+					<clay:content-row>
+						<clay:content-col>
+							<span class="sticker sticker-document sticker-rounded sticker-secondary">
```

Comment:
¿Is this `span` with all those classes needed?

Reply:
We need the `sticker-secondary` one, but the other classes are removable :+1: